netci.groovy

```diff
@@ -75,6 +75,25 @@ commitPullList.each { isPr ->
   }
 }
 
+// Windows CoreCLR
+commitPullList.each { isPr ->
+  ['debug', 'release'].each { configuration ->
+    def jobName = Utilities.getFullJobName(projectName, "windows_coreclr_test", isPr)
+    def myJob = job(jobName) {
+      description("Windows CoreCLR unit tests")
+            steps {
+              batchFile(""".\\build\\scripts\\cibuild.cmd ${(configuration == 'debug') ? '/debug' : '/release'} /skipDesktopTests""")
```

Comment:
This is why I dislike the /skip approach.  Reading this it's unclear why we arnet' also skipping VSI.